code/modules/clothing/glasses/hud.dm

```diff
@@ -65,6 +73,13 @@
 		emagged = 1
 		desc = desc + " The display flickers slightly."
 
+/obj/item/clothing/glasses/hud/security/sunglasses/emag_act(mob/user)

```

Comment:
This is handled by the /obj/item/clothing/glasses/hud/emag_act() already so it shouldn't be needed.